website/src/categories/components/badge/usage.md

```diff
@@ -61,17 +66,49 @@ Badges serve as visual cues to convey information without requiring user interac
 
 </section>
 
+<section>
+
+## Emphasis
+Understanding when to use subtle and bold emphasis for badge components is crucial for effective user interface design. Badges serve as visual cues to convey various types of information, ranging from minor details to critical alerts.
+
+<section class="ds-cards">
+  <figure>
+    <div class="ds-example">
+      <sl-icon name="fas-hand-back-point-up" aria-describedby="interactiveexample" size="sm"></sl-icon>
+      <sl-button fill="solid" variant="primary" size="lg">Solid</sl-button>
+    </div>
+    <figcaption>
+
+  ### Subtle (default)
+  Use subtle emphasis when the information conveyed by the badge is important but not critical. This could include supplementary details, status indicators, or minor distinctions.
+</figcaption>
+  </figure>
+  <figure>
+    <div class="ds-example">
+      <sl-icon name="fas-hand-back-point-up" aria-describedby="interactiveexample" size="sm"></sl-icon>
+      preview subtle badge
+    </div>
+    <figcaption>
+
+### Bold
+Reserve bold emphasis for situations where the information conveyed by the badge is crucial or demands immediate attention. This could include urgent notifications, critical alerts, or high-priority indicators.
+
+
+</section>
+
```

Comment:
I think another `</section>` closing is missing here.

Reply:
Fixed